### ISSUE TITLE
Throws when using `copyTpl` and .JPG

### ISSUE THREAD

**Issue by lukeed** [yeoman/generator#973]
When **copying** files from `templatePath` to `destinationPath`, I need to explicitly ignore all `.jpg` files (and presumably gif & png too):

```js
this.fs.copyTpl([
	`${this.templatePath()}/**`,
	'!**/test.js',
	'!**/*.jpg'
], this.destinationPath(), {...});
```

If I don't, the following stack is thrown:

```
Error: Could not find matching close tag for "<%".
    node_modules/ejs/lib/ejs.js:560:19
    Object.Template.generateSource (node_modules/ejs/lib/ejs.js:545:15)
    Object.Template.compile (node_modules/ejs/lib/ejs.js:450:12)
    Object.compile (node_modules/ejs/lib/ejs.js:306:16)
    handleCache (node_modules/ejs/lib/ejs.js:169:18)
    Object.exports.render (node_modules/ejs/lib/ejs.js:332:10)
    copy.extend.process (node_modules/mem-fs-editor/lib/actions/copy-tpl.js:12:18)
    applyProcessingFunc (node_modules/mem-fs-editor/lib/actions/copy.js:13:16)
    EditionInterface.exports._copySingle (node_modules/mem-fs-editor/lib/actions/copy.js:51:16)
    EditionInterface.exports.copy (node_modules/mem-fs-editor/lib/actions/copy.js:23:17)
    EditionInterface.module.exports [as copyTpl] (node_modules/mem-fs-editor/lib/actions/copy-tpl.js:10:8)
    cp (app/index.js:81:13)
    app/index.js:92:4
```

I know I can successfully copy images to my destination using `copy()`, but it can only be done one file at a time && makes the copy-templates phase clunkier.

```js
this.fs.copy(this.templatePath('assets/foo.jpg'), this.destinationPath('assets/foo.jpg'));
this.fs.copy(this.templatePath('assets/bar.jpg'), this.destinationPath('assets/bar.jpg'));
this.fs.copy(this.templatePath('assets/baz.jpg'), this.destinationPath('assets/baz.jpg'));
```

#### TL;DR;

Can we filter & ignore extensions within `fs.copyTpl` so that images can be cloned along with everything else in one fell swoop?

**Comment by lukeed** [comment]
Closing as I realize this belongs to [`mem-fs-editor`](https://github.com/SBoudrias/mem-fs-editor) instead.
